ros2param/test/test_verb_dump.py

```diff
@@ -16,17 +16,22 @@
 import os
 import tempfile
 import threading
+import time
 import unittest
 from unittest.mock import patch
+import xmlrpc
 
 import rclpy
 from rclpy.executors import MultiThreadedExecutor
 from rclpy.parameter import PARAMETER_SEPARATOR_STRING
 
 from ros2cli import cli
+from ros2cli.node.strategy import NodeStrategy
 
 TEST_NODE = 'test_node'
-TEST_NAMESPACE = 'foo'
+TEST_NAMESPACE = '/foo'
```

Comment:
Why did the test namespace need to change?

Reply:
`node.get_service_names_and_types_by_node` complains if we don't provide a namespace, whereas `rclpy.create_node` doesn't (the only other place `TEST_NAMESPACE` is used). I'm pretty sure at the `rcl` level, we need to provide a FQN for the namespace, `rclpy.create_node` is probably appending the missing forward slash for us.